configs/14.0/release_notes/relfixes.html

```diff
@@ -1,16 +1,83 @@
+<li>POWER10 enablement.</li>
 <li>POWER9 enablement.</li>
 <li>POWER9 optimized scheduler.</li>
 <li>Requires at least POWER8 for ppc64le.</li>
 <li>POWER8 Transactional Memory enablement.</li>
 <li>POWER8 Crypto Operations enablement.</li>
-<li>POWER8 and POWER9 optimized system libraries.</li>
+<li>POWER8, POWER9 and POWER10 optimized system libraries.</li>
```

Comment:
There's a debate about the "oxford comma", but my preference is to add a comma after "POWER9" here.

Reply:
Done